.storybook/preview.tsx

```diff
@@ -35,17 +39,19 @@ const globalStyles = css`
   ${baseStyle}
 `;
 
-export const decorators = [
-  renderStory => (
-    <EmotionCacheProvider>
-      <ThemeProvider>
-        <Global styles={globalStyles} />
-        <CssVariables />
-        {renderStory()}
-      </ThemeProvider>
-    </EmotionCacheProvider>
-  ),
-];
+export const decorators = isEmbeddingSDK
+  ? [] // No decorators for Embedding SDK stories, as we want to simulate real use cases
```

Comment:
Same as before, for the SDK we need to be on a clean state, simulating the customer app